index.bs

```diff
@@ -6264,6 +6264,37 @@ This attestation statement format is exclusively used by Apple for certain types
     5. Verify that the [=credential public key=] equals the Subject Public Key of |credCert|.
     6. If successful, return implementation-specific values representing attestation type [=Anonymization CA=] and attestation trust path |x5c|.
 
+## Compound Attestation Statement Format ## {#sctn-compound-attestation}
+
+The "compound" attestation statement format is used to pass multiple, self-contained attestation statements in a single ceremony.
+
+
+: Attestation statement format identifier
+:: compound
+
+: Attestation types supported
+:: Any attestation type defined in the IANA "WebAuthn Attestation Statement Format Identifiers" registry [[!IANA-WebAuthn-Registries]] except for `compound`
```

Comment:
The registry is not of attestation types, but attestation statement formats. This should probably just say "any"?

```suggestion
: Attestation types supported
:: Any. See [[#sctn-attestation-types]].
```